Golems/Lore.js

```diff
@@ -262,6 +262,14 @@ var lore = {
 		},
 		"pun7" : {
 			"text" : "PundexOutOfBounds - commence charging my PunReInator."
+		},
+		"pun8" : {
+			"text" : "No."
+		},
+		"pun9" : {
+			"text" : "A wild Artock appeared."

```

Comment:
missing closing } on pun9
Also please remove mentions of out of game real people, so the entire pun is either to be removed or changed

Reply:
I've added the } and changed Pun 9. Assumedly I don't know if this updates the pull request.